node.gyp

```diff
@@ -477,20 +516,14 @@
           ],
         }],
         [ 'OS=="freebsd" or OS=="linux"', {
-          'ldflags': [ '-Wl,-z,noexecstack',
+          'ldflags': [ '-Wl,-z,noexecstack,--allow-multiple-definition',
```

Comment:
Without it there are linking errors related to OpenSSL and V8, example:

`/out/Release/obj.target/deps/openssl/libopenssl.a(bio_ssl.o): In function 'BIO_f_ssl':
bio_ssl.c:(.text+0xb00): multiple definition of 'BIO_f_ssl'
/out/Release/libopenssl.a(bio_ssl.o):bio_ssl.c:(.text+0xb00): first defined here
/out/Release/obj.target/deps/openssl/libopenssl.a(bio_ssl.o): In function 'BIO_new_ssl':
bio_ssl.c:(.text+0xb10): multiple definition of 'BIO_new_ssl'
/out/Release/libopenssl.a(bio_ssl.o):bio_ssl.c:(.text+0xb10): first defined here
/out/Release/obj.target/deps/openssl/libopenssl.a(bio_ssl.o): In function 'BIO_new_ssl_connect':
bio_ssl.c:(.text+0xba0): multiple definition of 'BIO_new_ssl_connect'
/out/Release/libopenssl.a(bio_ssl.o):bio_ssl.c:(.text+0xba0): first defined here
/out/Release/obj.target/deps/openssl/libopenssl.a(bio_ssl.o): In function 'BIO_new_buffer_ssl_connect':
bio_ssl.c:(.text+0xc10): multiple definition of 'BIO_new_buffer_ssl_connect'
/out/Release/libopenssl.a(bio_ssl.o):bio_ssl.c:(.text+0xc10): first defined here
/out/Release/obj.target/deps/openssl/libopenssl.a(bio_ssl.o): In function 'BIO_ssl_copy_session_id':
bio_ssl.c:(.text+0xc90): multiple definition of 'BIO_ssl_copy_session_id'
`